src/test/java/ru/ayaz/VideoTest.java

```diff
@@ -0,0 +1,64 @@
+package ru.ayaz;
+
+import org.junit.Test;
+
+import java.io.File;
+
+import static org.junit.Assert.*;
+import static org.mockito.Mockito.mock;
+
+public class VideoTest {
+
+    @Test
+    public void shouldContainCommentInListWhenCommentAdded() {
```

Comment:
&#x1F34E; **Issue**
Also about video not being just a container of comments:
shouldContainCommentInListWhenCommentAdded -> shouldHaveCommentWhenCommentAdded

And again "InList" - which list ? It looks more like an implementation detail. Domain model doesn't have a list related to a video (you could probably create an entity called CommentsList, but its rather unnecessary right now).